src/always-encrypted/cek-table.ts

```diff
@@ -0,0 +1,62 @@
+import { EncryptionKeyInfo } from './types';
+
+export class CEKTable {
```

Comment:
Do we need this class, or could we just use an array instead to hold CEK values?

Reply:
It's not needed. I've removed it and replaced it with an array as you suggested